descriptors/ModuleDescriptor-template.json

```diff
@@ -159,6 +159,13 @@
           "permissionsRequired": [ "bulk-operations.download.item.get" ],
           "modulePermissions": [
           ]
+        },
+        {
+          "methods": [ "GET" ],
+          "pathPattern": "/bulk-operations/list-users",
+          "permissionsRequired": ["bulk-operations.list-users.collection.get"],
+          "modulePermissions": [
```

Comment:
@obozhko-folio , the userClient.getUserById() is executed in the endpoint logic, so I think some additional permissions should be in modulePermissions section. Could you please check this.

Reply:
Good catch, added permission.